pkg/operator/operator.go

```diff
@@ -472,7 +472,7 @@ func addFinalizer(instance *v1alpha1.Driver, f string) bool {
 	return true
 }
 
-func removeFinalizer(instance *v1alpha1.Driver, f string) {
+func removeFinalizer(instance *v1alpha1.AWSEBSDriver, f string) {
```

Comment:
nit: the finalizer is called `csi.storage.operator.openshift.io`

Reply:
Fixes (as a separate commit)